messages/shape.create.md

```diff
@@ -1,16 +1,20 @@
 # summary
 
-Create a scratch org configuration (shape) based on the specified source org
+Create a scratch org configuration (shape) based on the specified source org.
 
 # description
 
-Create a scratch org configuration (shape) based on the specified source org.
+Scratch org shapes mimic the baseline setup (features, limits, edition, and Metadata API settings) of a source org without the extraneous data and metadata.
+
+Run "<%= config.bin %> org list shape" to view the available org shapes and their IDs.
+
+To create a scratch org from an org shape, include the "sourceOrg" property in the scratch org definition file and set it to the org ID of the source org. Then create a scratch org with the "<%= config.bin %> force:org:create" command.
```

Comment:
@mshanemc Is the ref to `force:org:create` is intentional?

Reply:
yes, at least for a while until we get that plugin done.